<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>RebootViewerBase</class>
 <widget class="QWidget" name="RebootViewerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Reboot Cluster</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QLabel" name="mTitleLbl" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="autoFillBackground" >
      <bool>true</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth" >
      <number>3</number>
     </property>
     <property name="text" >
      <string>Reboot Cluster</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="mRebootClusterLbl" >
       <property name="text" >
        <string>Set Cluster Boot Target:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mSelectWinBtn" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../MaestroResource.qrc" >:/Maestro/images/win_xp.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mSelectLinuxBtn" >
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../MaestroResource.qrc" >:/Maestro/images/linux2.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="mNodeTableView" >
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="mRebootBtn" >
       <property name="toolTip" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:Sans Serif; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Reboot Entire Cluster&lt;/p>&lt;/body>&lt;/html></string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="../MaestroResource.qrc" >:/Maestro/images/reboot.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../MaestroResource.qrc" />
  <include location="ClusterSettingsResource.qrc" />
 </resources>
 <connections/>
</ui>
